### PR TITLE
insert/update stmt

### PR DESCRIPTION
**Pull request by Mulavar**
**What this PR does**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: pkg/runtime/xxast/xxast.go [hunk 1]
```diff
@@ -96,37 +97,84 @@ func FromStmtNode(node ast.StmtNode) (Statement, error) {
...
+			OrderBy:    nOrderBy,
+			Limit:      nLimit,
 		}, nil
 	}
 	return nil, nil
 }
 
 func convInsertColumns(columnNames []*ast.ColumnName) []string {
-	var result = make([]string, len(columnNames))
+	var result = make([]string, 0)
```
Comment (jjeffcaii) [inline pkg/runtime/xxast/xxast.go:177]: Use `make([]string, 0, len(columnNames))` instead, will reduce slice allocation.

File: pkg/runtime/xxast/xxast.go [hunk 2]
```diff
@@ -224,13 +272,58 @@ func (cc *convCtx) convFieldList(node *ast.FieldList) []SelectElement {
 	return ret
 }
 
+// convert assignment, like a = 1.
+func (cc *convCtx) convAssignment(assignments []*ast.Assignment) []*UpdateElement {
+	var result []*UpdateElement
```
Comment (jjeffcaii) [inline pkg/runtime/xxast/xxast.go:277]: Same as above

File: pkg/runtime/xxast/xxast.go [hunk 3]
```diff
@@ -224,13 +272,58 @@ func (cc *convCtx) convFieldList(node *ast.FieldList) []SelectElement {
...
+		})
+	}
+	return result
+}
+
+func (cc *convCtx) convOrderBy(ob *ast.OrderByClause) OrderByNode {
+	if ob == nil {
+		return nil
+	}
+	var result OrderByNode
```
Comment (jjeffcaii) [inline pkg/runtime/xxast/xxast.go:303]: Same as above.

**Review by AlexStocks — APPROVED** [review]

**Review by dongzl — APPROVED** [review]

**Review by Lvnszn — APPROVED** [review]

File: pkg/runtime/xxast/xxast.go [hunk 4]
```diff
@@ -224,13 +272,58 @@ func (cc *convCtx) convFieldList(node *ast.FieldList) []SelectElement {
...
+		var nColumn ColumnNameExpressionAtom
+		column := assignment.Column
+		if column.Schema.O != "" {
+			nColumn = append(nColumn, column.Schema.O)
+		}
+		if column.Table.O != "" {
+			nColumn = append(nColumn, column.Table.O)
+		}
+		if column.Name.O != "" {
+			nColumn = append(nColumn, column.Name.O)
```
Comment (cjphaha) [inline pkg/runtime/xxast/xxast.go:288]: 这里可以这样 column.Name.String() 和  column.Name.O 等效的

**Review by cjphaha — APPROVED** [review]

**Pull request merged**